### ISSUE TITLE
Complete /faq/ section (Google Seasons of Docs)

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps.github.io#211]
#### Problem

Many users have questions regarding the functionality, features, and usage of the Organic Maps mobile app. Without a comprehensive F.A.Q. section, users may encounter difficulties in understanding and utilizing the app effectively, leading to frustration and reduced user satisfaction.

#### Scope

The scope of this task involves creating an end-user F.A.Q. section that addresses common questions, concerns, and inquiries related to the Organic Maps mobile app. The F.A.Q. should cover various aspects of the app, including but not limited to installation, navigation, features, settings, troubleshooting, and support.

#### Success Criteria

1. **Comprehensive Coverage**: The F.A.Q. should cover a wide range of topics and address common user queries comprehensively.
2. **Clarity and Conciseness**: Each question and corresponding answer should be clear, concise, and easy to understand for users of all levels of technical proficiency.
3. **Accessibility**: The F.A.Q. should be easily accessible within the app, preferably through a dedicated section in the app's menu or settings.
4. **Updated Information**: Ensure that the F.A.Q. reflects the latest version of the Organic Maps app, including any recent updates or changes.
5. **User Feedback Integration**: Consider incorporating feedback from user testing and previous user inquiries to address common pain points and improve the overall effectiveness of the F.A.Q.

#### Skills

- **Technical Writing**: Ability to communicate complex technical concepts in a clear and understandable manner suitable for end-users.
- **User Experience (UX) Design**: Understanding of user needs and preferences to create an intuitive and user-friendly F.A.Q. layout and navigation.
- **Knowledge of Organic Maps**: Familiarity with the features, functionalities, and settings of the Organic Maps mobile app to accurately address user inquiries.
- **Communication**: Ability to collaborate with stakeholders, including developers, designers, and product managers, to gather relevant information and ensure alignment with the app's objectives and user needs.
- **Quality Assurance**: Thorough testing and validation of the F.A.Q. content to ensure accuracy, completeness, and effectiveness in addressing user queries.

#### See also

- [Existing FAQ in the app](https://github.com/organicmaps/organicmaps/blob/master/data/faq.html)
- [Initial draft of the F.A.Q. section for Web](https://github.com/organicmaps/organicmaps.github.io/pull/185)
- [Add App FAQs PR](https://github.com/organicmaps/organicmaps.github.io/pull/204)
- [Import FAQ from the main repo](https://github.com/organicmaps/organicmaps.github.io/pull/115)

**Comment by biodranik** [comment]
Important note: the FAQ should be compilable into offline HTML pages to be distributed with the app, and be able to load images remotely if internet access is enabled/available.

If compiled images do not take much space, they can even be distributed with the app.

**Comment by Ayush23Dash** [comment]
I would be interested to work on this, could I get a communication channel link to discuss this and introduce myself!?

**Comment by Ayush9026** [comment]
Hi @rtsisyk and @biodranik sir 

I'm interested in tackling the FAQ issue. I'll compile it into offline HTML for easy access and ensure remote image loading when internet is available. Let's enhance user experience together.

**Comment by Ayush9026** [comment]
@rtsisyk  Absolutely, my primary focus will be on content creation and documentation improvement during Google Season of Docs. However, if there's additional time available or in the future, I'm open to contributing to design enhancements or any other areas that could further enhance the user experience. My main goal is to ensure that the FAQ documentation is comprehensive, clear, and easily accessible to users. Thank you for clarifying the focus of the program, and I'm eager to collaborate on enhancing the content.

**Comment by KARTIKEY752** [comment]
@rtsisyk  Hi ,i would like to work on this project .

**Comment by IIITM-Jay** [comment]
Hi @rtsisyk and @biodranik,  I have spent time on drafting a proposal which I have shared via email.
Let me know your thoughts and will love to hear the feedback on the same.
